cliquet/tests/test_cache.py

```diff
@@ -156,12 +159,86 @@ def test_ping_returns_false_if_unavailable(self):
     def test_ping_logs_error_if_unavailable(self):
         pass
 
+    def test_cache_prefix_is_set(self):
+        settings_prefix = self.settings.copy()
+        settings_prefix['cache_prefix'] = 'prefix_'
+        config_prefix = self._get_config(settings=settings_prefix)
+
+        # initiating cache backend with prefix:
+        backend_prefix = self.backend.load_from_config(config_prefix)
+
+        # set a value with a cache that has no prefix
+        backend_prefix.set('key', 'foo')
+
+        # obtain the value of cache that has prefix
+        obtained = backend_prefix.get('key')
+
+        self.assertEqual(obtained, 'foo')
+
+    def test_cache_when_prefix_is_not_set(self):
+        settings_prefix = self.settings.copy()
+        config_prefix = self._get_config(settings=settings_prefix)
+        backend_prefix = self.backend.load_from_config(config_prefix)
+
+        # set a value with a cache that has no prefix
+        backend_prefix.set('key', 'foo')
+
+        obtained = backend_prefix.get('key')
+        self.assertEqual(obtained, 'foo')
+
+    def test_prefix_value_use_to_delete_data(self):
+        settings_prefix = self.settings.copy()
+        settings_prefix['cache_prefix'] = 'prefix_'
+        config_prefix = self._get_config(settings=settings_prefix)
+
+        # initiating cache backend with prefix:
+        backend_prefix = self.backend.load_from_config(config_prefix)
+
+        # set a value with a cache that has no prefix
+        backend_prefix.set('key', 'foo')
+
+        # deleting data
+        backend_prefix.delete('key')
+
+    def test_prefix_value_used_with_ttl(self):
+        settings_prefix = self.settings.copy()
```

Comment:
Since you are doing this code in all the test cases it probably worth to put in a ``def setUp(self)`` method.